torch/_inductor/triton_heuristics.py

```diff
@@ -422,7 +423,7 @@ def launcher({', '.join(def_args)}, grid, stream):
         launcher.config = cfg
         launcher.n_regs = getattr(binary, "n_regs", None)
         launcher.n_spills = getattr(binary, "n_spills", None)
-        launcher.shared = getattr(binary, "shared", None)
```

Comment:
Wondering why not just `s/binary/scope/` here?

Reply:
I think we doesn't really need to run an extra search via ```getattr```. Seems assigning a previously-retrieved value is slightly more straightforward :)